packages/wallet-service/tests/integration.test.ts

```diff
@@ -152,44 +152,6 @@ afterAll(async () => {
   process.env = OLD_ENV;
 });
 
-// eslint-disable-next-line jest/prefer-expect-assertions, jest/expect-expect
-test('receive blocks and txs and then start wallet', async () => {
```

Comment:
We don't have tests for this, need to create tests that use both the lambdas and the daemon